library/core/src/marker.rs

```diff
@@ -316,7 +316,7 @@ pub trait StructuralEq {
 /// attempt to derive a `Copy` implementation, we'll get an error:
 ///
 /// ```text
-/// the trait `Copy` may not be implemented for this type; field `points` does not implement `Copy`
+/// the trait `Copy` must not be implemented for this type; field `points` does not implement `Copy`
```

Comment:
keep this unchanged, unless you're going to modify the error message of E0204 too.

```
error[E0204]: the trait `Copy` may not be implemented for this type
 --> src/lib.rs:1:10
  |
1 | #[derive(Copy, Clone)]
  |          ^^^^
2 | struct X {
3 |     a: &'static mut i32,
  |     ------------------- this field does not implement `Copy`
  |
  = note: this error originates in the derive macro `Copy` (in Nightly builds, run with -Z macro-backtrace for more info)
```